src/serumVial.ts

```diff
@@ -49,7 +47,7 @@ export class SerumVialClient {
       }
     };
 
-    this.ws.onclose = (ev) => {
+    this.ws.onclose = (_ev) => {
```

Comment:
what is this?

Reply:
unused variables dont complain to lint if they start with an underscore

The argument ev is passed into the callback, I think it stands for event, but doesnt matter since we dont use it